components/google_sheets/actions/add-single-row/add-single-row.mjs

```diff
@@ -1,10 +1,44 @@
 import googleSheets from "../../google_sheets.app.mjs";
 
+const dataFormat = {
+  type: "string",
+  label: "Data Format",
+  description: "You may enter individual values for each column or provide a single array representing the entire row.",
+  options: [
+    {
+      label: "Enter a value for each column",
+      value: "column",
+    },
+    {
+      label: "Pass an array of values",
+      value: "array",
+    },
+  ],
+  reloadProps: true,
+};
+
+const headerProp = {
+  type: "string",
+  label: "Header Row?",
+  description: "If the first row of your document has headers we'll retrieve them to make it easy to enter the value for each column.",
+  options: [
+    {
+      label: "First row has headers",
+      value: "hasHeaders",
+    },
+    {
+      label: "There is no header row",
+      value: "noHeaders",
+    },
+  ],
+  reloadProps: true,
+};
+
 export default {
   key: "google_sheets-add-single-row",
   name: "Add Single Row",
   description: "Add a single row of data to Google Sheets",
-  version: "0.1.2",
+  version: "1.0.0",
```

Comment:
```suggestion
  version: "0.2.0",
```

Reply:
@dannyroosevelt Is it okay changing the major?

Reply:
> @dannyroosevelt Is it okay changing the major?

yes major version is actually ideal in this scenario since this update will not be backward compatible w/ existing workflows. so if you update the component in your workflow, we'll prompt users to update the props, etc in order to save.